safe-hts-precompile/SafeOperations.sol

```diff
@@ -0,0 +1,330 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import "./SafeHTS.sol";
+
+contract SafeOperations {
+
+    function safeTokenAssociate(address sender, address tokenAddress) external {
```

Comment:
You can keep the same names for all methods as they are in the SafeHTS library, e.g. `safeAssociateToken`